zcash_proofs/src/lib.rs

```diff
@@ -22,18 +22,20 @@ pub mod prover;
 
 pub fn load_parameters(
     spend_path: &Path,
-    spend_hash: &str,
     output_path: &Path,
-    output_hash: &str,
     sprout_path: Option<&Path>,
-    sprout_hash: Option<&str>,
 ) -> (
     Parameters<Bls12>,
     PreparedVerifyingKey<Bls12>,
     Parameters<Bls12>,
     PreparedVerifyingKey<Bls12>,
     Option<PreparedVerifyingKey<Bls12>>,
 ) {
+    // Sapling circuit hashes
+    const SAPLING_SPEND_HASH: &str = "8270785a1a0d0bc77196f000ee6d221c9c9894f55307bd9357c3f0105d31ca63991ab91324160d8f53e2bbd3c2633a6eb8bdf5205d822e7f3f73edac51b2b70c";
+    const SAPLING_OUTPUT_HASH: &str = "657e3d38dbb5cb5e7dd2970e8b03d69b4787dd907285b5a7f0790dcc8072f60bf593b32cc2d1c030e00ff5ae64bf84c5c3beb84ddc841d48264b4a171744d028";
+    const SPROUT_HASH: &str = "e9b238411bd6c0ec4791e9d04245ec350c9c5744f5610dfcce4365d5ca49dfefd5054e371842b3f88fa1b9d7e8e075249b3ebabd167fa8b0f3161292d36c180a";
```

Comment:
I verified that all three hashes match the [equivalent hashes in `zcashd`](https://github.com/zcash/zcash/blob/d52003861c9c2a846c8ffceb07805da446414fe8/src/init.cpp#L729-L739).